docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
+  queryParameters:
+    cmd:
+      description: the command of this application
+      example: java
+    id:
+      description: the id or part of the id of the application
+      example: /us-east/database/memsql
+    label:
+      description:
+        A label selector query contains one or more label selectors, which are comma separated.
+        Marathon supports three types of selectors existence-based, equality-based and set-based.
+        In the case of multiple selectors, all must be satisfied so comma separator acts as an AND logical operator.
+        Labels and values must consist of alphanumeric characters plus `-` `_` and `.` `-A-Za-z0-9_.`.
+        Any other character is possible, but must be escaped with a backslash character.
+
+        * <code>Existence based Selector Query</code> Matches the existence of a label
+
+        * <code>Equality based Selector Query</code> Matches existence of labels and the (non) equality of the value.
+
+        * <code>Set based Selector Query</code> Matches existence of labels and the (non) existence of the value in a given set
+
+      example: environment==production, tier!=frontend\ tier, deployed in (us, eu), deployed notin (aa, bb)
```

Comment:
An example for an existence based selector is missing; only equality and set based are given. As far as I understand this, setting label to `environment` would match apps that have an `environment` set – no matter to which value.